qa/rpc-tests/feature_fedpeg.py

```diff
@@ -49,12 +49,15 @@ def setup_network(self, split=False):
                 rpc_u, rpc_p = rpc_auth_pair(n)
                 self.extra_args.append([
                     "-printtoconsole=0",
+                    "-addresstype=legacy", # To make sure bitcoind gives back p2pkh no matter version
+                    "-deprecatedrpc=validateaddress",
+                    # Parent chain is looking for its own conf(e.g., bitcoin.conf)
+                    # These are not generated or cached during batch functional testing
+                    # so we enter the required lines here
```

Comment:
This wording is confusing. They are needed, because the generated config files are called `elements.conf` and bitcoind looks for `bitcoin.conf`. I didn't understand that out of your explanation :laughing_smiley_with_sweat_drop:

Reply:
bitcoin.conf isn't generated. What's confusing?

Reply:
Maybe I misinterpreted the wording :p Still, since it's in `parent_bitcoin == True`, "Parent chain is looking for its own conf(e.g., bitcoin.conf)" could be "bitcoind looks for bitcoin.conf, which is not generated..." or  something. But nvm, I think I just got confused with the "cached" and looking into the `--cachedir` for the Travis thing.

Reply:
i didnt feel like fixing up this comment if we end up adding more chain support in the future :)